src/labels/CustomLabels.labels

```diff
@@ -8432,6 +8432,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description for Connect Two Contacts with Relationships checklist item</shortDescription>
+        <value>Relationships in NPSP let you to capture important connections between your constituents. Connect Contacts using the Relationships related list on the Contact records.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactsRelationshipsLink</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link text for Connect Two Contacts with Relationships checklist item</shortDescription>
+        <value>Learn more about Relationships in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactsRelationshipsTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title for Connect Two Contacts with Relationships checklist item in Get Started</shortDescription>
+        <value>Connect Two Contacts with Relationships</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCreateContactAndAccountBlueButton</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Blue button text for Create Contacts and Account checklist item in Get Started</shortDescription>
+        <value>Create Contacts</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCreateContactAndAccountDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description for Create Contacts and Account checklist item in Get Started page</shortDescription>
+        <value>People are the most important resource for a nonprofit, so understanding how we use the Contact object to track them in NPSP is key. The Contact object stores constituent data such as name, address, and employer.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCreateContactAndAccountLink</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link text for Contact and Accounts checklist item in Get Started page</shortDescription>
+        <value>Learn more about Contact and Accounts in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCreateContactAndAccountTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title for Create Contacts and Account checkbox item in Get Started page</shortDescription>
```

Comment:
Please change to: Title for Contacts and Accounts checklist item in Get Started.